src/components/EditInput/index.jsx

```diff
@@ -0,0 +1,139 @@
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { updateMyInfo } from "../../apis/api";
+
+export const MyPageInput = ({ content }) => {
+  const [isEmailEdit, setIsEmailEdit] = useState(false);
+  const [isUsernameEdit, setIsUsernameEdit] = useState(false);
+  const [isCollegeEdit, setIsCollegeEdit] = useState(false);
+  const [isMajorEdit, setIsMajorEdit] = useState(false);
```

Comment:
이 부분의 경우 `formData`를 처리하는 것처럼 한번에 관리해주는게 좋아요!
```suggestion
  const [isEdit, setIsEdit] = useState({ 
  email : false, 
  username: false, 
  college: false, 
  major: false
  })
```

이런식으로요!